src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationFeedbackPresenter.cs

```diff
@@ -142,6 +130,7 @@ public void Handle(ProjectClosedEvent eventToHandle)
          clearFeedbackReferences();
          resetFeedback();
          _view.NoFeedbackAvailable();
+         _view.Hide();
```

Comment:
Why do you had the view? the user won't see that the PI is finished or even see the optimal results etc... or am i missing something?

Reply:
If you close the project, should we keep the view open? To me, closing the project means you are finished with everything on it, including checking if the PI has finished or not. Actually, the references will be cleared meaning you should not be able to keep the view opened, right?

Reply:
yeah ok. makes sense. What was the current behaviour? It would just stay open?